ui/components/MeshConfigComponent.js

```diff
@@ -879,12 +941,71 @@ handleNATSClick = () => {
             variant="contained"
             color="primary"
             size="large"
-            onClick={() => this.handleReset()}
+            onClick={handleClickOpenDatabaseDialogue}
             className={classes.button}
             data-cy="btnResetDatabase"
           >
               Reset Database
           </Button>
+          <Dialog
```

Comment:
@Harshit-Dandriyal could you please use this pre-defined Prompt component from Meshery UI, to maintain consistency across modals. https://github.com/meshery/meshery/blob/master/ui/components/PromptComponent.js

Reply:
Thanks @Chadha93, Done :+1: